src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -221,7 +221,11 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
           </Skeleton>
           <Collapse startingHeight={70} in={showDescription}>
             <SkeletonText isLoaded={isLoaded} noOfLines={4} spacing={2} skeletonHeight='20px'>
-              <SanitizedHtml color='gray.500' dirtyHtml={description} />
+              <SanitizedHtml
+                color='gray.500'
+                dirtyHtml={description}
+                isTrustedDescription={isTrustedDescription}
```

Comment:
I would prefer using more of a generalized `isTrusted` instead of `isTrustedDescription` for the param so the `SanitizedHtml` component can be reused for other needs.